docs/source/chatops/chatops.rst

```diff
@@ -251,5 +251,5 @@ Logging
 =======
 
 ChatOps logs are written to ``/var/log/st2/st2chatops.log`` on non systemd-based distros. For
-systemd-based distros (Ubuntu 18/20, RHEL/RockyLinux8+), you can access the logs via
+systemd-based distros (Ubuntu 18+, RHEL/RockyLinux8+), you can access the logs via
```

Comment:
Further down you state that 18.04 focal is no longer supported, but here this shows 18+, should it just be `20+`